setup.py

```diff
@@ -47,8 +47,6 @@
 Intended Audience :: Developers
 License :: OSI Approved :: Zope Public License
 Programming Language :: Python
-Programming Language :: Python :: 2
-Programming Language :: Python :: 2.7
```

Comment:
If removing these, we should probably add:
`Programming Language :: Python :: 3 :: Only`